src/noisepy/seis/S1_fft_cc_MPI.py

```diff
@@ -70,7 +70,9 @@ def cross_correlate(
     # #########PROCESSING SECTION##########
```

Comment:
I would consider converting to a block comment for readability. For example:

```
"""
PROCESSING SECTION
"""
```

Reply:
Yeah, there's a lot of style and formatting work remaining in this code base, but not part of this PR